cloudsql-mysql-plugin/src/e2e-test/features/sink/CloudMySqlDesignTime.feature

```diff
@@ -0,0 +1,36 @@
+@CloudMySql
+Feature: CloudMySql sink- Verify CloudMySql sink plugin design time scenarios
+
+  Scenario: To verify CloudMySql sink plugin validation with mandatory properties
```

Comment:
There are more scenarios in the RTM where we are validating the plugin with different properties present in the plugin. Create more Design Time scenarios .

Reply:
I don't see this one addressed.